README.rst

```diff
@@ -258,6 +258,34 @@ For accounts in regions outside of US-West, use :code:`region` parameter to spec
     $dbh = new PDO("snowflake:account=testaccount.us-east-1", "user", "password");
     $dbh = new PDO("snowflake:account=testaccount;region=us-east-1", "user", "password");
 
+Using Key Pair Authentication
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The PHP PDO driver supports key pair authentication and key rotation.
+
+You must first complete the initial configuration for key pair authentication as shown in 
+`Key Pair Authentication & Key Pair Rotation <https://docs.snowflake.com/en/user-guide/key-pair-auth.html#key-pair-authentication-key-pair-rotation>`_.
+
+To connect to the Snowflake database using key par authentication, create a new :code:`PDO` object, as explained in the
+`PHP PDO documentation <https://www.php.net/manual/en/pdo.connections.php>`_.
+Specify the data source name (:code:`dsn`) parameter as shown below:
+
+.. code-block:: php
+
+    $dbh = new PDO("snowflake:account=<account_name>", 
+                   "AUTHENTICATOR = SNOWFLAKE_JWT", 
+                   "PRIV_KEY_FILE = <path>/rsa_key.p8`",
+                   "PRIV_KEY_FILE_PWD = <password>");
+
+where:
+
+- :code:`<account_name>` is
+  `your Snowflake account name <https://docs.snowflake.com/en/user-guide/connecting.html#your-snowflake-account-name>`_.
+- :code:`AUTHENTICATOR = SNOWFLAKE_JWT` Specifies to authenticate the Snowflake connection using key pair authentication with JSON Web Token (JWT).
+- :code:`PRIV_KEY_FILE = <path>/rsa_key.p8` Specifies the local path to the private key file you created (i.e. **rsa_key.p8**).
+- :code:`PRIV_KEY_FILE_PWD = <password>` Specifies the passcode to decode the private key file.
```

Comment:
As shown in https://docs.snowflake.com/en/user-guide/key-pair-auth.html#step-1-generate-the-private-key , we make a distinction between support for encrypted private keys and support for unencrypted private keys.

Do we also support unencrypted private keys? (in which case you'd omit the passphrase for the private key) Or do we only support encrypted private keys?

Reply:
We support both.

Adding verbiage to indicate it is optional for unencrypted keys